synapse/handlers/device.py

```diff
@@ -27,6 +29,21 @@ class DeviceHandler(BaseHandler):
     def __init__(self, hs):
         super(DeviceHandler, self).__init__(hs)
 
+        self.hs = hs
```

Comment:
I've been doing ```self.is_mine_id = hs.is_mine_id``` in new code in a mostly futile attempt to avoid having ``self.hs`` scattered throughout the handlers.